packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -138,9 +139,25 @@ export class VaButton {
     if (this.submit === undefined) {
       return;
     }
-    // eslint-disable-next-line i18next/no-literal-string
-    const theForm = this.el.closest('form');
-    if (!theForm) {
+
+    // Find the form element by walking up through shadow roots
+    let element: Element = this.el;
+    let formElement: HTMLFormElement | null = null;
+    
+    while (element && !formElement) {
```

Comment:
This is the meat of the update. When the `va-button` web component is used as a child of another web-component (ie. va-button-pair), a `form` element was not being found by simply looking for `this.el.closest('form');`.

This update will expand the search for a `form` element up through multiple shadow roots. When it finds one, it will use that for submitting the form. If it doesn't find one, it'll break out of the while loop and return without submitting or dispatching anything